mozetl/taar/taar_lite_guidguid.py

```diff
@@ -0,0 +1,189 @@
+"""
+This ETL job computes the co-installation occurrence of white-listed
+Firefox webextensions for a sample of the longitudinal telemetry dataset.
+"""
+
+import click            # noqa
+import boto3            # noqa
+import datetime as dt   # noqa
+import json             # noqa
+import logging          # noqa
+from botocore.exceptions import ClientError   # noqa
+from pyspark.sql import Row, SparkSession     # noqa
+from pyspark.sql.functions import col, collect_list, explode, udf, sum as sum_, max as max_, first  # noqa
+from pyspark.sql.types import StructType, StructField, StringType, LongType, IntegerType  # noqa
+from taar_utils import store_json_to_s3
+from taar_utils import load_amo_external_whitelist
+
+logging.basicConfig(level=logging.INFO)
+log = logging.getLogger(__name__)
+
+OUTPUT_BUCKET = 'telemetry-parquet'
+OUTPUT_BUCKET = 'telemetry-parquet'
+OUTPUT_PREFIX = 'taar/lite/'
+OUTPUT_BASE_FILENAME = 'guid_coinstallation'
+
+AMO_DUMP_BUCKET = 'telemetry-parquet'
+AMO_DUMP_KEY = 'telemetry-ml/addon_recommender/addons_database.json'
+MAIN_SUMMARY_PATH = 's3://telemetry-parquet/main_summary/v4/'
+ONE_WEEK_AGO = (dt.datetime.now() - dt.timedelta(days=7)).strftime('%Y%m%d')
+
+
+def extract_telemetry(spark):
+    """ load some training data from telemetry given a sparkContext
+    """
+    sc = spark.sparkContext
+
+    # Define the set of feature names to be used in the donor computations.
+
+    def get_initial_sample(pct_sample=10):
+        # noqa: ignore=E127,E502,E999
```

Comment:
I would try to limit the amount of `#noqa`, it's generally not a great sign.

Reply:
I got rid of this noqa flag as flake8 didn't seem to care when I removed it. There was a flake8 long line error because of spark syntax that I've patched in dd18f1dfc7f03fa92230f77a74bb3dfa24a36f05